website/src/content/deployment-examples/google-cloud-storage.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: Deploying to Google cloud
+---
+
+This deployment example refers to [Google cloud Static Web Apps](https://codelabs.developers.google.com/codelabs/cloud-webapp-hosting-gcs#0)
+
+# Prerequisites
+
+Before you get started, you need to have:
+
+- A [Google](https://accounts.google.com/SignUp) account.
+- A [valid domain](https://codelabs.developers.google.com/codelabs/cloud-webapp-hosting-gcs#1) that you own or manage.- A [commercetools](https://docs.commercetools.com/getting-started) account and a Project.
```

Comment:
```suggestion
- A [valid domain](https://codelabs.developers.google.com/codelabs/cloud-webapp-hosting-gcs#1) that you own or manage.
- A [commercetools](https://docs.commercetools.com/getting-started) account and a Project.
```

Reply:
Thank you. Updated here: https://github.com/commercetools/merchant-center-application-kit/pull/2740/commits/c197b5db5d2ae0c53e15e3b05abc4661e2b4cb21